src/middlewared/middlewared/alembic/versions/24.10/2024-08-16_10-01_lagg_on_delete_cascade.py

```diff
@@ -0,0 +1,34 @@
+"""Cascade delete LAGG interface configuration when network interface configuration is deleted
```

Comment:
`alias` and `bridge` table have this, there is no sense for `lagg` table not to have it.